lib/createProfileDbData.js

```diff
@@ -0,0 +1,12 @@
+export default function createProfileDbData(userInput, imageUrl) {
+  const dbData = {
+    userName: "Kevin Ketner",
+    bikeName: userInput.bikeName,
+    bikeType: userInput.type,
+    bought: userInput.bought,
+    mileage: userInput.mileage,
+    lastService: userInput.lastService,
+    profileImg: imageUrl,
+  };
```

Comment:
since I know you are a friend of not repeating yourself. It should work like this as well:

```suggestion
const {bikeName, type: bikeType, bought, mileage, lastService} = userInput;

  const dbData = {
    userName: "Kevin Ketner",
    bikeName,
    bikeType,
    bought,
    mileage,
    lastService,
    profileImg: imageUrl,
  };
```

Reply:
That's a nice one. Looks so much better. 😍 I've implemented all your suggestions. Thank's again for your advices😃